Library/Formula/libcutl.rb

```diff
@@ -0,0 +1,24 @@
+class Libcutl < Formula
+  homepage "http://www.codesynthesis.com/projects/libcutl/"
+  url "http://www.codesynthesis.com/download/libcutl/1.9/libcutl-1.9.0.tar.gz"
+  sha256 "1b575aa8ed74aa36adc0f755ae9859c6e48166a60779a5564dd21b8cb05afb7d"
+
+  depends_on "gcc" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  fails_with :clang do
+    cause <<-EOS.undent
+    Oh! It seems that you only have clang available, or GCC wasn't found!
```

Comment:
Theoretically, you'll need to specify at least the core system compilers for the OS X versions Homebrew supports - Otherwise on the older OS X versions they don't have Clang by default so Homebrew will try to build with ` LLVM ` or even Apple's old ` gcc `. I think ` fails_with :clang ` and ` fails_with :llvm ` would achieve the same thing as a hard ` gcc ` dep, but run it past @mikemcquaid - Compilers are one of my pet hates.